spec/index.bs

```diff
@@ -6698,7 +6693,19 @@ dictionary GPUCommandEncoderDescriptor : GPUObjectDescriptorBase {
 
             **Returns:** {{GPUCommandEncoder}}
 
-            Issue: Describe {{GPUDevice/createCommandEncoder()}} algorithm steps.
+            1. Let |e| be a new {{GPUCommandEncoder}} object.
+            1. Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    1. If any of the following conditions are unsatisfied
+                        [$generate a validation error$], make |e| [=invalid=], and stop.
+                        <div class=validusage>
+                            - |this| is [=valid=].
```

Comment:
Something to discuss eventually is whether we actually ever need to check this. In principle if the device is lost, we can't observe anything that happens on the "device timeline" (not even validation errors) because it doesn't exist. So all of these checks might actually be redundant? It would make #1629 easier to reason about also.